plugin/src/main/kotlin/com/nishtahir/CargoBuildTask.kt

```diff
@@ -42,6 +44,10 @@ open class CargoBuildTask : DefaultTask() {
                 ?: targetDirectory
                 ?: "${module!!}/target"
 
+            if (Files.exists(Paths.get(targetDirectory))) {
```

Comment:
This conditional looks backwards -- surely you want `!Files.exists(...)`.

Reply:
You're right. Thanks!